0002-correct-install-path-and-remove-uninstall-executable.patch

```diff
@@ -0,0 +1,38 @@
+From 467069edad11afa61bd1f2f7873f091fc44dcd8e Mon Sep 17 00:00:00 2001
+From: Martin Marmsoler <martin.marmsoler@gmail.com>
+Date: Wed, 2 Nov 2022 15:41:18 +0100
+Subject: [PATCH 2/2] correct install path and remove uninstall executable,
+ because it is not needed for flatpak
+
+---
+ source6/ChartGeany/ChartGeany.pro | 13 +++++--------
+ 1 file changed, 5 insertions(+), 8 deletions(-)
+
+diff --git a/source6/ChartGeany/ChartGeany.pro b/source6/ChartGeany/ChartGeany.pro
+index 39aad23..a172b1a 100644
+--- a/source6/ChartGeany/ChartGeany.pro
++++ b/source6/ChartGeany/ChartGeany.pro
+@@ -129,15 +129,12 @@ QMAKE_CXXFLAGS += -fomit-frame-pointer -fstack-protector
+ # QMAKE_STRIP = echo
+ QMAKE_POST_LINK += strip $(TARGET)
+ 
+-target.path = /opt/$$PKG_NAME
+-images.path = /opt/$$PKG_NAME
+-desktop.path = /usr/share/applications/
+-images.files = images/$${PKG_NAME}.png
++target.path = $${PREFIX}/bin
++desktop.path = $${PREFIX}/share/applications/
+ desktop.files = $${PKG_NAME}.desktop
+-chartgeany.path = /opt/$$PKG_NAME
+-chartgeany.commands = rm -rf /usr/bin/$$TARGET; ln -s /opt/$$PKG_NAME/$$TARGET /usr/bin
+-chartgeany.uninstall = rm -rf /usr/bin/$$TARGET
+-INSTALLS += target desktop images chartgeany
++metainfo.path = $${PREFIX}/share/metainfo/
++metainfo.path = $${PGK_NAME}.metainfo.xml
++INSTALLS += target desktop metainfo
```

Comment:
you could as well also patch to install the icons in the right place.

Reply:
I did not find a way to do it properly from the .pro file with qmake